src/auth.ts

```diff
@@ -169,85 +182,183 @@ export function initBaseAuth(opts: BaseAuthOptions) {
         fetchOrgByQuery: fetchOrgsByQueryWrapper,
         fetchUsersByQuery: fetchUsersByQueryWrapper,
         fetchUsersInOrg: fetchUsersInOrgWrapper,
+        // user management functions
         createUser: createUserWrapper,
         updateUserMetadata: updateUserMetadataWrapper,
         updateUserEmail: updateUserEmailWrapper,
         createMagicLink: createMagicLinkWrapper,
         migrateUserFromExternalSource: migrateUserFromExternalSourceWrapper,
-        createOrg: createOrgWrapper,
-        addUserToOrg: addUserToOrgWrapper,
         deleteUser: deleteUserWrapper,
         disableUser: disableUserWrapper,
         enableUser: enableUserWrapper,
+        // org management functions
+        createOrg: createOrgWrapper,
+        addUserToOrg: addUserToOrgWrapper,
         allowOrgToSetupSamlConnection: allowOrgToSetupSamlConnectionWrapper,
         disallowOrgToSetupSamlConnection: disallowOrgToSetupSamlConnectionWrapper,
-        UserRole,
     }
 }
 
+// wrapper function with no validation
 function wrapValidateAccessTokenAndGetUser(tokenVerificationMetadataPromise: Promise<TokenVerificationMetadata | void>) {
     return async function validateAccessTokenAndGetUser(authorizationHeader?: string): Promise<User> {
         return extractAndVerifyBearerToken(tokenVerificationMetadataPromise, authorizationHeader)
     }
 }
 
-function wrapValidateAccessTokenAndGetUserWithOrg(tokenVerificationMetadataPromise: Promise<TokenVerificationMetadata | void>) {
-    const validateAccessTokenAndGetUserWithOrgInfo = wrapValidateAccessTokenAndGetUserWithOrgInfo(tokenVerificationMetadataPromise)
+// The following eight functions are wrappers around our four validations: isRole, atLeastRole, hasRequirement, hasAllRequirements
```

Comment:
The orgId functions were kept around for backwards compatibility reasons. Given that this is a major release, we can cut those 4 functions and have people migrate over to the orgInfo versions.